app/Repositories/Admin/Album/AlbumRepository.php

```diff
@@ -21,4 +21,9 @@ public function deleteSongOfAlbum($id)
     {
         return $this->find($id)->songs()->delete();
     }
+
+    public function getAlbumsFromAuthors(array $authorIds)
+    {
+        return $this->model->whereIn('author_id', $authorIds)->get();
+    }
```

Comment:
function này em có thể viết vào repository chung nè, truyền vào key và list of values